src/layers/UsePathsLayer.tsx

```diff
@@ -73,6 +78,58 @@ function addUnselectedPathsLayer(map: Map, paths: Path[]) {
     map.addInteraction(select)
 }
 
+function createBezierLineString(start: number[], end: number[]):LineString {
+    var curvePoints = [];
+    var center = [(start[0] + end[0]) / 2, (start[1] + end[1]) / 2];
+    var radius = distance(start, end) / 2;
+
+    var startAngle = Math.atan2(start[1] - center[1], start[0] - center[0]);
+    var endAngle = Math.atan2(end[1] - center[1], end[0] - center[0]);
+
+    // Define the control points for the Bezier curve
+    var controlPoints = [
+        center[0] + 1/2 * radius * Math.sin(startAngle + 1/2 * (endAngle - startAngle)),
+        center[1] + 1/2 * radius * Math.cos(startAngle + 1/2 * (endAngle - startAngle))
+    ];
+
+    // Calculate intermediate points along the curve using a Bezier curve
+    curvePoints.push(start);
+    for (var t = 0; t <= 1; t += 0.1) {
+        var point = [
+            (1-t)*(1-t)*start[0] + 2*t*(1-t)*controlPoints[0] + t*t*end[0],
+            (1-t)*(1-t)*start[1] + 2*t*(1-t)*controlPoints[1] + t*t*end[1]
+        ];
+        curvePoints.push(point);
+    }
+    curvePoints.push(end)
+    return new LineString(curvePoints);
+}
+
+function addAccessNetworkLayer(map: Map, selectedPath: Path, queryPoints: QueryPoint[]) {
+    const style = new Style({
+        stroke: new Stroke({
+            // color: 'rgba(170,170,170,1)',
+            color: '#275DAD',
+            width: 4,
+            lineDash: [1, 10],
+            lineCap: 'round',
+            lineJoin: 'round'
+        }),
+    })
+    var layer = new VectorLayer({
+        source: new VectorSource()
+    });
+    layer.setStyle(style);
+    for(let i=0;i<selectedPath.snapped_waypoints.coordinates.length;i++){
```

Comment:
For consistent formatting you can run `npm run format` or integrate the prettier configuration with your IDE or text editor.

Reply:
Oh ja, thanks for the hint. Just did it.

Reply:
Du kannst das natürlich auch mergen, oder soll ich?

Reply:
Mach lieber du. Ich bin hier nur Gast.

Reply:
Danke @easbar

Reply:
Ein willkommener Gast. Danke dir!